src/Engine/Tables/NPCTable.h

```diff
@@ -6,19 +6,32 @@
 #include "Engine/Objects/NPCEnums.h"
 
 #include "Utility/IndexedArray.h"
+#include "Utility/Flags.h"
+
+// TODO(Nik-RE-dev): It seems that two greet flags are used purely because it's modification is performed
+//                   before greeting string is constructed. It is also ensures that NPC in multi-NPC houses
+//                   always greet you with first line until you leave the house.
+//                   Ideally there should be only one flag.
+enum class NPC_FLAG : uint32_t {
```

Comment:
Actually, our codestyle is:
* Enums are types, and thus are named as types. In this particular case, it's `NpcFlag`.
* Enum values are named in `SNAKE_CASE`.